test/tensor/Jamfile

```diff
@@ -62,4 +61,7 @@ test-suite boost-ublas-tensor-test
           test_fixed_rank_tensor_matrix_vector.cpp
           unit_test_framework 
     ]
+    [ run test_functions.cpp
+          unit_test_framework
+    ]
```

Comment:
The files `test_function.cpp` and `test_multiplication.cpp` include `static_extents`. Would it be better to split those files, one that is only heaving static extents?

Reply:
This was created during `GSoC` so forgot about it. I was also thinking the same thing when I realized this was causing the crash in `msvc`. I will refactor it.